### PR TITLE
Add crystal library helm chart

### PR DESCRIPTION
**Pull request by iamvigneshwars**

### CODE REVIEW

File: charts/xchemlab/charts/crystal-library/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.0.1
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "0.0.5"
```
Comment (garryod) [inline charts/xchemlab/charts/crystal-library/Chart.yaml:15]: These all need bumped with your latest release, can be done as a separate commit / PR though

File: charts/xchemlab/charts/crystal-library/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v2
+name: compound-library
```
Comment (garryod) [inline charts/xchemlab/charts/crystal-library/Chart.yaml:2]: Surely `crystal-library`?

**Review by garryod — CHANGES_REQUESTED** [review]

File: charts/xchemlab/charts/crystal-library/Chart.yaml (hunk 1 above)
Comment (iamvigneshwars) [reply charts/xchemlab/charts/crystal-library/Chart.yaml]: I've changed them in a separate commit.

File: charts/xchemlab/charts/crystal-library/Chart.yaml (hunk 2 above)
Comment (iamvigneshwars) [reply charts/xchemlab/charts/crystal-library/Chart.yaml]: Changed it.

**Review by garryod — APPROVED** [review]

**Pull request merged**